source/CIPlanetNode.h

```diff
@@ -17,7 +17,7 @@
 #include "CIPlanetLayer.h"
 
 #define PLANET_RING_TEXTURE_INNER_SIZE 140
-#define PLANET_OUTER_RING_SCALE 0.833
+#define PLANET_OUTER_RING_SCALE 1.770
```

Comment:
Can you change line 96 in PlnaetNode.cpp to be the same as PlanetNode.h? The initial creation of the node outer ring should use this scale the same way that setRadius does

Reply:
So replace the value passed in line 96 with "PLANET_OUTER_RING_SCALE * _layerScale/_coreScale"?